plugins/base/frontend/src/main/components/navigationPaneSearch/navigationPaneSearch.scss

```diff
@@ -0,0 +1,37 @@
+@import "src/main/scss/index.scss";
+
+$defaultHeight: 40px;
+
+div#paneSearch {
+
+    width: 248px;
+    margin: 0 auto;
+
+    input#navigation-pane-search {
+        background: $white;
+        border: 1px solid $grey-border;
+        box-sizing: border-box;
+        border-radius: 4px;
+        padding: 8px;
+        height: $defaultHeight;
+    }
+    
+    .navigation-pane-search {
+        width: 100% !important;
```

Comment:
Is this `!important` really necessary?

Reply:
Tbh i don't see another solution. The issue is that this component requires me to provide a min-width value in pixels (by default 240px), so i can make it exactly 248px (280px - paddings) but i'd much rather tell it to be 100% of the container it is in. The `!important` part is caused by the fact, that this value in pixels is provided to this component via styles in html so it is impossible to provide a more specific selector.